src/app/footer/footer.component.html

```diff
@@ -1,18 +1,21 @@
-<footer>
-    <div class="links d-flex col-8">
-        <a href="https://www.linkedin.com/in/yuriyshchebetun/" class="badge">
-            <i class="fab fa-linkedin-in"></i>
-        </a>
-        <a href="https://github.com/Yuriy-27" class="badge">
-            <i class="fab fa-github"></i>
-        </a>
-        <a href="https://t.me/YuriyShchebetun" class="badge">
-            <i class="fab fa-telegram-plane"></i>
-        </a>
-        <a href="skype:yuriyshchebetun?userinfo" class="badge">
-            <i class="fab fa-skype"></i>
-        </a>
-    </div>
-    <p>2019 &copy; Created by Yuriy Shchebetun</p>
+<div class="footer container-fluid">
+        <div class="links d-flex row">
+            <a href="https://www.linkedin.com/in/yuriyshchebetun/" class="badge">
```

Comment:
@Yuriy-27  please use *ngFor looping through an array with objects where you can store `link` and `iconClass` fields